app/components/PerformanceMetrics.vue

```diff
@@ -1,37 +1,41 @@
 <template>
   <div class="performance-metrics flex">
-    <span class="performance-metric-wrapper resolution">
-      <i class="performance-metric-icon icon-display"/>
+    <span class="performance-metric-wrapper resolution" v-if="!compactMode">
```

Comment:
```suggestion
    <span class="performance-metric-wrapper resolution" v-if="shouldShowResolution">
```

のように要素ごとに表示するか判定するメソッドを用意する形にしておくと出し分け条件が理解しやすく変更にも強くなるかなと思いました
ただ、同様の箇所が大量にあるので今対応する必要はないと思います

Reply:
これは一旦後日か...